app/controllers/posts_controller.rb

```diff
@@ -1,14 +1,40 @@
 class PostsController < ApplicationController
```

Comment:
- [ ] [REQUIRED] Kindly ensure that you add a `new` action in the Post controller in order to create a new post. 
Currently, I can't access the route for the form to create a new post.

- Kindly check the review summary for the remaining recommended changes
<img width="677" alt="Screenshot 2023-11-01 at 19 07 02" src="https://github.com/AlyAtalla/blog-app/assets/58692590/ddf5de0e-a042-41fd-a9fb-85ae5e0d7044">